handlers/handlers.go

```diff
@@ -89,14 +111,92 @@ func HandleSessionRequest(
 			panic(err)
 		}
 
+		// Create token as a cookie and set it
+		{
+			userId, err := userservice.GetUserIdFromEmailAddress(loginForm.EmailAddress)
+			if err != nil {
+				panic(err)
+			}
+
+			token, err := createTokenAsString(userId, oneWeekInMinutes)
+			if err != nil {
+				panic(err)
+			}
+
+			expiration := time.Now().Add(oneWeekInMinutes * time.Minute)
+
+			cookie := http.Cookie{
+				Name:     cerealNotesCookieName,
+				Value:    token,
+				Expires:  expiration,
+				HttpOnly: true,
+			}
+
+			http.SetCookie(responseWriter, &cookie)
+
+		}
+
 		responseWriter.WriteHeader(http.StatusCreated)
 		responseWriter.Write([]byte(fmt.Sprint("passward email combo was correct")))
 
+	case http.MethodDelete:
+		//Cookie will overwrite existing cookie then delete itself
```

Comment:
Well we need to delete the cookie. And since we prevent javascript from accessing the cookie using `HttpOnly:true`. I don't know how to delete it from the client side, so I think we'll need the domain to delete the cookie.

Reply:
got it. never knew about httpOnly. Only ever fucked with cookies in js